documentation/modules/oauth/con-oauth-authentication-client-options.adoc

```diff
@@ -3,33 +3,34 @@
 // assembly-oauth-authentication.adoc
 
 [id='con-oauth-authentication-client-options-{context}']
-= OAuth 2.0 client authentication flow
+= OAuth 2.0 client authentication flows
 
-In this section, we explain and visualize the communication flow between Kafka client, Kafka broker, and authorization server during Kafka session initiation.
-The flow depends on the client and server configuration.
+[role="_abstract"]
+OAuth 2.0 authentication flows depend on the underlying Kafka client and Kafka broker configuration.
+The flows must also be supported by the authorization server used.
 
-When a Kafka client sends an access token as credentials to a Kafka broker, the token needs to be validated.
+Kafka broker listener configuration determines how clients authenticate using an access token.
+The client can pass a client ID and secret to request an access token.
+If a listener is configured to use PLAIN authentication, you can use a client username and password to request or pass access tokens.
```

Comment:
This doesn't sound right. there is no such thing as 'client username'. There is either `clientId` or `username`. The way it sounds here is as if we support password authentication, where a user can configure the client with some user account's username and password. That is not supported even when PLAIN authentication is used. What is supported is to authenticate with clientId and secret or with username and access token and passing these as `username` and `password` parameters of the SASL/PLAIN authentication mechanism.  
```suggestion
If a listener is configured to use PLAIN authentication, PLAIN mechanism can be used on the client to authenticate with `clientId` and `secret`, or `username` and `access token` by passing these as `username` and `password` parameters of the PLAIN mechanism.
```

Reply:
I've updated as follows:

> If a listener is configured to use PLAIN authentication, the client can authenticate with a client ID and secret or username and access token. These values are passed as the `username` and `password` properties of the PLAIN mechanism.

Reply:
Sounds better, yes.